content/concepts/content-routing.md

```diff
@@ -3,6 +3,112 @@ title: Content Routing
 weight: 5
 ---
 
-This article is coming soon!
+Peer-to-peer networks are inherently tricky to route because of the following 
+interdisciplinary routing challenges:
 
-Please [refer to this issue](https://github.com/libp2p/docs/issues/23) to track the progress and make suggestions.
+- The lack of universal orchestration that a central server can provide when 
+  querying and retrieving content.
+- Not having a central directory that contains information about reaching every peer 
+  in the network.
+- The presence of high node churn.
+- Data classification is communication-expensive.
+- Creating a resilient, scalable, and optimal routing protocol that is resistant to 
+  Sybil attacks and node churn while being future-proof.
```

Comment:
A lot of this is restating the previous points, I'd split this into two bullet points:

* Sybil attacks
* Forward compatibility